src/Octoshift/Services/ReclaimService.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Octoshift.Models;
+using OctoshiftCLI;
+using OctoshiftCLI.Models;
+
+namespace Octoshift.Services
+{
+    public class ReclaimService
+    {
+        private readonly GithubApi _githubApi;
+        private readonly OctoLogger _log;
+        private bool _failed;
+
+        public ReclaimService(GithubApi githubApi, OctoLogger logger)
+        {
+            _githubApi = githubApi;
+            _log = logger;
+        }
+        public virtual async Task ReclaimMannequin(string mannequinUser, string mannequinId, string targetUser, string githubOrg, bool force)
+        {
+            var githubOrgId = await _githubApi.GetOrganizationId(githubOrg);
+
+            var mannequins = new Mannequins((await GetMannequins(githubOrgId)).GetByLogin(mannequinUser, mannequinId));
```

Comment:
`Mannequins` is a service and needs to be injected. If the intention is to group some helper methods that are only meant to be used in this service then you can put them here as private methods also. But otherwise all services need to be properly injected and registered in the DI container.

Reply:
I think it's more of a model, how do you consider it a service?

Reply:
This is by definition a service b/c it provide bunch of functionality via methods to another service. Whereas a model is an entity with bunch of fields.

Reply:
Again if you feel like you don't want to inject it then it's a indicator that you need to create bunch of private methods instead of the service class.

Reply:
Injecting seems more effort than it's worth it.

So either inline everything into reclaim service and make it more complicate or make this an nested class of reclaim service.

Reply:
How about just moving the methods to the service class as private methods? Creating a nested class seems like an overkill to me but it also works.

Reply:
nested class added